Userland/Libraries/LibCodeComprehension/Cpp/Tests.cpp

```diff
@@ -89,9 +89,9 @@ int run_tests()
 
 static void add_file(FileDB& filedb, DeprecatedString const& name)
 {
-    auto file = Core::DeprecatedFile::open(LexicalPath::join(TESTS_ROOT_DIR, name).string(), Core::OpenMode::ReadOnly);
-    VERIFY(!file.is_error());
-    filedb.add(name, DeprecatedString::copy(file.value()->read_all()));
+    auto file_or_error = Core::File::open(LexicalPath::join(TESTS_ROOT_DIR, name).string(), Core::File::OpenMode::Read);
+    VERIFY(!file_or_error.is_error());
```

Comment:
Isn't that the same as `realese_value_but_fixme_should_propagate_errors`?

Reply:
It's inside a test, so there's no real destination where to report an error. So it isn't really a FIXME, because it would still abort the test. I don't really like the idea of adding an unfixable fixme. We could make it a "MUST", what do you think about that?

And just in advance: `TRY_OR_FAIL` doesn't work here because this test suite wasn't written in the "return ErrorOr" style. Introducing it now would require much more work, and doesn't fit into a PR about DeprecatedFile migration.